packages/xo-vmdk-to-vhd/src/ova.js

```diff
@@ -342,7 +342,7 @@ export async function parseOVAFile(
           return parseGzipFromEnd(start, end, fileSlice, header)
         }
       }
-      data.tables[header.fileName] = await readVmdkGrainTable(readFile)
+      data.tables[header.fileName] = readVmdkGrainTable(readFile)
```

Comment:
A promise that can reject must always have a `catch` listener (or `await`) added before the next async loop.

If that's the case here, you can do this:

```suggestion
      const promise = readVmdkGrainTable(readFile)
      promise.catch(noop) // prevent unhandled rejection
      data.tables[header.fileName] = promise
```

Reply:
do I really need to do that? I'd be happy for any error to be thrown as an exception when the `await` will block on this promise.

Reply:
That will still be the case: the rejection will always be part of the promise and observable.

But you can't have a rejected promise without a listener at any given time

Reply:
ok, I will do the fix.

Reply:
I did it inside `readVmdkGrainTable()`

Reply:
I don't think that's the right place for it, it's too hidden.

Also, it does not work because the `await` in `readVmdkGrainTable` can throw which will reject the returned promise (which does not have a reject listener).